package.json

```diff
@@ -9,6 +9,7 @@
   "author": "Vivek Patel <patelvivek@google.com>",
   "bugs": "https://github.com/oysterbooks/oy/issues",
   "main": "lib/Oy.js",
+  "types": "./src/index.d.ts",
```

Comment:
Could we move the index.d.ts file to the top-level? It should make discovery easier for new users.

Reply:
We definitely can!

In a TypeScript project, it'd be pretty common for the TS compiler to generate the .d.ts file under the `/src` directory, next to the sources, and generate the built .js files under the `/lib` directory. So this setup is definitely not uncommon, and so long as the package.json file points to it then it'll be discovered by TS compilers and intellisense. But really it's pretty arbitrary, so whatever you're happy with :)

Reply:
Thanks for the context. I was trying to find a few examples and saw a mix of locations. I've personally found the type files helpful to quickly understand the API of a project, and so thought the top-level would be best.